src/onion/client.md

```diff
@@ -4,7 +4,7 @@ This is a description of onion layer in Tox.
 
 Onion routing can be described using this chart:
 
-![](./static/path.svg)
```

Comment:
how about / instead of ./ ?

Reply:
That would resolve to `https://tox-rs.github.io/` instead of `https://tox-rs.github.io/book/`.

Reply:
Did you try?

Reply:
Yes. `/book/static/path.svg` works, as expected.

Reply:
```
-![](https://tox-rs.github.io/book/static/path.svg)
+![](../static/path.svg)
```
This works too